ProcessMembership/handler.ts

```diff
@@ -390,6 +390,10 @@ const submitMembershipClaimToSelfcare = (selfcareClient: SelfCareClient) => (
             )
           )
     ),
+    TE.mapLeft(
+      error =>
+        new Error(`${error.message} , Request Body : ${JSON.stringify(claim)}`)
```

Comment:
I think with `|` as separating character it is more readable
```suggestion
        new Error(`${error.message} | Request Body : ${JSON.stringify(claim)}`)
```